interface.idl

```diff
@@ -116,5 +116,12 @@ namespace rtc {
 namespace time {
     rpc gettimeofday() -> (unixtime: uint64);
 }
-
+namespace shm {
+    rpc create(size : size) ->  (shm_id : int);
+    rpc map(shm_id : int, flag:int)   ->  (vaddr : vaddr);
```

Comment:
In this patch, `flag` indicates whether the mapped area is writable, right? If so, let's rename this field to `writable: bool`.